projects/models.py

```diff
@@ -48,54 +65,55 @@ class Project(models.Model):
         related_name="projects_catalog",
         help_text="Catalog id that this project uses",
     )
-    IMPACT_LEVEL_CHOICES = [
-        ("low", "Low"),
-        ("moderate", "Moderate"),
-        ("high", "High"),
-        ("pii/phi", "PII or PHI"),
-    ]
+    catalog_version = models.CharField(
+        max_length=64,
+        default="CMS ARS 3.1",
+        blank=False,
+        help_text="The Catalog version, for example, NIST 800-53r5",
+    )
     impact_level = models.CharField(
-        choices=IMPACT_LEVEL_CHOICES,
+        choices=ImpactLevel.choices,
         max_length=20,
-        default=None,
-        null=True,
+        null=False,
+        blank=False,
         help_text="FISMA impact level of the project",
     )
-    LOCATION_CHOICES = [
-        ("cms_aws", "CMS AWS Commercial East-West"),
-        ("govcloud", "CMS AWS GovCloud"),
-        ("azure", "Microsoft Azure"),
-        ("other", "Other"),
-    ]
     location = models.CharField(
-        choices=LOCATION_CHOICES,
+        choices=LocationChoices.choices,
         max_length=100,
         default=None,
         null=True,
         help_text="Where the project is located",
     )
-    STATUS_CHOICES = [
-        ("active", "Active"),
-        ("archived", "Archived"),
-    ]
     status = models.CharField(
-        choices=STATUS_CHOICES,
+        choices=StatusChoices.choices,
         max_length=20,
-        default="active",
+        default=StatusChoices.ACTIVE,
         help_text="Status of the project",
     )
     created = models.DateTimeField(auto_now_add=True, db_index=True)
     updated = models.DateTimeField(auto_now=True, db_index=True, null=True)
 
-    # Additional permissions to manage members.
-    # (Add, change, delete, and view permissions are automatically created)
     class Meta:
+        """
+        Additional permissions to manage members.
+        (Add, change, delete, and view permissions are automatically created)
+        """
+
         permissions = [
             ("can_add_members", "Can add members"),
             ("can_delete_members", "Can delete members"),
             manage_project_users_permission,
         ]
 
+    def save(self, *args, **kwargs):
+        version = getattr(self, "catalog_version")
+        impact = getattr(self, "impact_level")
+        if version and impact:
+            catalog = Catalog.objects.get(version=version, impact_level=impact)
```

Comment:
Need to handle the cases where multiple objects are returned or a matching object isn't found. Otherwise this will result in a 500 with an unhelpful message